### ISSUE TITLE
prince.PCA vs. sklearn.decomposition.PCA?

### ISSUE THREAD

**Issue by normanius** [MaxHalford/prince#151]
I'm comparing the PCA functionality from sklearn ([sklearn.decomposition.PCA](https://scikit-learn.org/stable/modules/generated/sklearn.decomposition.PCA.html)) with the one from prince. I cannot fully understand why the results differ. 

**Example:**

```python
import prince
data = prince.datasets.load_energy_mix(year=2019, normalize=True)

from sklearn.decomposition import PCA
pca_sklearn = PCA(n_components=2)
pca_sklearn = pca_sklearn.fit(data)

pca_prince = prince.PCA(n_components=2)
pca_prince = pca_prince.fit(data)
```

I was looking at `pca_sklearn.singular_values_` or `pca_sklearn.explained_variance_ratio_`, but could not relate those values to `pca_prince.eigenvalues_` or `pca_prince.cumulative_percentage_of_variance_`. 

Aren't these methods supposed to be equivalent?

**Comment by MaxHalford** [comment]
Hey there 👋

The trick is that you have to normalize the data with scikit-learn. The following works:

```py
import prince
data = prince.datasets.load_energy_mix(year=2019, normalize=True)

from sklearn import pipeline, preprocessing, decomposition

pca_prince = prince.PCA(n_components=2)
pca_prince = pca_prince.fit(data)

pca_sklearn = pipeline.make_pipeline(
    preprocessing.StandardScaler(),
    decomposition.PCA(n_components=2),
)
pca_sklearn = pca_sklearn.fit(data)
```

```py
>>> pca_sklearn[-1].singular_values_
array([12.45303013, 11.10384117])

>>> pca_prince.svd_.s
array([12.45303013, 11.10384117])

```

I suggest taking a look at the [PCA unit tests](https://github.com/MaxHalford/prince/blob/master/tests/test_pca.py), which test Prince against scikit-learn.

**Comment by normanius** [comment]
Awesome! Thanks for the clarifications, and sorry for the noise.

**Comment by MaxHalford** [comment]
Don't apologize :)
